numba/unicode.py

```diff
@@ -1388,6 +1389,20 @@ def impl(s):
     return impl
 
 
+# https://github.com/python/cpython/blob/1d4b6ba19466aba0eb91c4ba01ba509acf18c723/Objects/unicodeobject.c#L11729-L11737    # noqa: E501
+if sys.version_info[:2] >= (3, 7):
+    @overload_method(types.UnicodeType, 'isascii')
+    def unicode_isalpha(data):
```

Comment:
```suggestion
    def unicode_isascii(data):
```
please be careful with naming.

Reply:
Fixed